core/playback/mpv/mpv.go

```diff
@@ -0,0 +1,141 @@
+package mpv
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/navidrome/navidrome/conf"
+	"github.com/navidrome/navidrome/log"
+)
+
+// mpv --no-audio-display --pause 'Jack Johnson/On And On/01 Times Like These.m4a' --input-ipc-server=/tmp/gonzo.socket
+const (
+	mpvComdTemplate = "mpv --no-audio-display --pause %f --input-ipc-server=%s"
+)
+
+func start(args []string) (Executor, error) {
+	log.Debug("Executing mpv command", "cmd", args)
+	j := Executor{args: args}
+	j.PipeReader, j.out = io.Pipe()
+	err := j._start()
+	if err != nil {
+		return Executor{}, err
+	}
+	go j.wait()
+	return j, nil
+}
+
+func (j *Executor) Cancel() error {
+	if j.cmd != nil {
+		return j.cmd.Cancel()
```

Comment:
Unfortunately Cancel() is not supported by Go 1.19, and Navidrome project is committed to support the two latest major Go versions (currently 1.20 and 1.19). Go 1.21 will be released next month. Can we live without this for the time being?

Reply:
Then I would vote to just wait for Go 1.21, since the Cancel() function is used for closing each single track. Otherwise I would have to downport a similar function ...

Reply:
OK Go 1.21 is released